packages/ng/core-select/user/users.directive.ts

```diff
@@ -130,7 +130,7 @@ export class LuCoreSelectUsersDirective<T extends LuCoreSelectUser = LuCoreSelec
 		const params = {
 			fields: this.#userFields,
 			...(this._filters() ?? {}),
-			...(this._operationIds() ? { operationIds: this._operationIds().join(',') } : {}),
+			...(this._operationIds() ? { operations: this._operationIds().join(',') } : {}),
```

Comment:
wasn't this fixed somewhere already? I'm having "déjà vu" :D

Reply:
it was fixed a few lines above, for regular options. The present fix is for the "Me: " option 🙈